base/notify-celery-main/celery-deployment.yaml

```diff
@@ -22,7 +22,8 @@ spec:
       labels:
         app: celery
         # profile: fargate
-    spec:
+    spec:  
+      priorityClassName: celery
```

Comment:
Does that mean that some celery workers could be in the same situation of 0 instances if nodes are reworked during a scaling procedure?

Reply:
In theory yes, but I think the risk is low because we only ever have 2 instances of things like admin or doc download running, we won't run into a scenario where we need to evict every celery pod from primaries (right now each primary node is running ~4 celery pods)

I'd like to give it a try this way, with Steve's tests. If it proves to be too unpredictable, I'll do the two deployment thing.

Reply:
sounds good.